File: design-patterns/go/composite/transport_list/car.go
// car.go

package main

import "fmt"

type Car struct {
}

func NewCar() (car *Car) {
	car = &Car{}
	return
}

func (car *Car) Start() {
	fmt.Println("Starting Car...")
}

func (car *Car) Stop() {
	fmt.Println("Stopping Car...")
}

func (car *Car) Operate() {
	fmt.Println("Driving Car")
}

File: design-patterns/php/composite/transportlist/Car.php
<?php
// Car.php

namespace BigBoxCode\DesignPattern\Composite\TransportList;


class Car implements Transport {
    public function start(): void {
        echo "Starting Car...\n";
    }

    public function operate(): void {
        echo "Driving Car\n";
    }

    public function stop(): void {
        echo "Stopping Car...\n";
    }
}